wrappers/python/examples/wnt/connections.py

```diff
@@ -0,0 +1,176 @@
+"""
+    Connections
+    ===========
+
+    WNT backend connections
+
+    .. Copyright:
+        Copyright 2019 Wirepas Ltd. All Rights Reserved.
+        See file LICENSE.txt for full license details.
+"""
+
+import ssl
+import logging
+import websocket
+import threading
+
+
+class Connections(object):
```

Comment:
We have ported this code to the backend client. These tests ought to be integrated there.